cmd/lncli/commands.go

```diff
@@ -621,7 +621,10 @@ var closeChannelCommand = cli.Command{
 	In the case of a cooperative closure, One can manually set the fee to
 	be used for the closing transaction via either the --conf_target or
 	--sat_per_byte arguments. This will be the starting value used during
-	fee negotiation. This is optional.`,
+	fee negotiation. This is optional.
+
+	To view which funding_txids can be used for a channel close,
+	see the channel_point values within the listchannels command output.`,
```

Comment:
should specify that `channel_point` is `funding_txid` + `output_index`

Reply:
Done!